docker/Dockerfile

```diff
@@ -86,10 +86,11 @@ RUN \
 	# Remove Git and clean apt cache
 	apt-get clean && \
 	apt-get remove -y git && \
-	apt-get -y autoremove
+	apt-get -y autoremove && \
+	chmod +x docker/entrypoint.sh
 
 USER node
 ENV NODE_ENV=production
 
-ENTRYPOINT ["pm2-runtime", "bin/iotagent-ul"]
+ENTRYPOINT ["docker/entrypoint.sh"]
```

Comment:
Relative path will work with all IoT Agents

Reply:
Same as https://github.com/telefonicaid/iotagent-json/pull/380#discussion_r258422568

Reply:
Response same as https://github.com/telefonicaid/iotagent-json/pull/380#discussion_r258503643
:smile: 

Again let's discuss on that thread first and I'll make changes later.

I believe `docker/entrypoint.sh` will be easier to maintain

Reply:
NTC according to https://github.com/telefonicaid/iotagent-json/pull/380#discussion_r259467292